pythonforandroid/recipes/py3dns/patches/android.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/DNS/Base.py b/DNS/Base.py
+index 34a6da7..15f83a2 100644
+--- a/DNS/Base.py
++++ b/DNS/Base.py
+@@ -49,9 +49,13 @@ defaults= { 'protocol':'udp', 'port':53, 'opcode':Opcode.QUERY,
+             'server': [] }
+ 
+ def ParseResolvConf(resolv_path="/etc/resolv.conf"):
+-    "parses the /etc/resolv.conf file and sets defaults for name servers"
+-    with open(resolv_path, 'r') as stream:
+-        return ParseResolvConfFromIterable(stream)
++    try:
++        "parses the /etc/resolv.conf file and sets defaults for name servers"
```

Comment:
It doesn't matter too much, but docstring could stay where it was instead of being in the try/except block

Reply:
done